services/web/server/src/simcore_service_webserver/security_roles.py

```diff
@@ -71,43 +72,14 @@
         ],
         "inherits": [UserRole.GUEST, UserRole.ANONYMOUS],
     },
-    UserRole.TESTER: {"can": [], "inherits": [UserRole.USER]},
+    UserRole.TESTER: {
+        "can": [],
+        "inherits": [UserRole.USER],
+    },
 }
 
-#
-# REFERENCE IN THE FRONT_END
-#
 
-# "anonymous": [],
-# "guest": [
-###   "studies.templates.read",
-#   "study.node.data.pull", , <----------???
-###   "study.start",
-###   "study.stop",
-###   "study.update"
-# ],
-# "user": [
-###   "studies.user.read",
-###   "studies.user.create",
-###   "storage.datcore.read",
-###   "user.user.update",
-###   "user.apikey.create",
-###   "user.apikey.delete",
-###   "user.token.create",
-###   "user.token.delete",
-###   "study.node.create",
-###   "study.node.delete",
-###   "study.node.rename",
-###   "study.node.start",
-#   "study.node.data.push", <----------???
-#   "study.node.data.delete", <----------???
-# XX   "study.edge.create",
-# XX   "study.edge.delete"
-# ],
-# "tester": [
-#   "services.all.read",   <----------???
-###   "user.role.update",
-#   "study.nodestree.uuid.read", <----------???
-#   "study.logger.debug.read" <----------???
-# ],
-# "admin": []
+# static test
+assert set(e for e in ROLES_PERMISSIONS) == set(  # nosec
+    e for e in UserRole
+), "All user rols must be part define permissions"  # nosec
```

Comment:
Why not have this in a test? From what I understand in non development mode this will not run.

Reply:
it is as well.. but here I use defensive programming for development and a simple import will tell me if I messed up the table